Documentation/BuildInstructions.md

```diff
@@ -111,7 +111,11 @@ Once you're in nix-shell, you should be able to follow the build directions.
 Make sure you have all the dependencies installed:
 
 ```console
-brew install coreutils qemu e2fsprogs m4 autoconf libtool automake bash gcc@10 ninja cmake
+# core
+brew install coreutils qemu bash gcc@10 ninja cmake
+
+# Fuse + ex2
```

Comment:
It's `ext2`, not `ex2` (also in commit message) :)

Reply:
Whoops, fixed!